subprojects/build-cache/src/integTest/groovy/org/gradle/caching/local/internal/DirectoryBuildCacheCleanupIntegrationTest.groovy

```diff
@@ -115,30 +116,30 @@ class DirectoryBuildCacheCleanupIntegrationTest extends AbstractIntegrationSpec
         // One day isn't enough to trigger
         when:
         // Set the time back 1 day
-        gcFile().setLastModified(originalCheckTime - TimeUnit.DAYS.toMillis(1))
+        gcFile().setLastModified(originalCheckTime - TimeUnit.HOURS.toMillis(1))
```

Comment:
How about we change the test name, too? To something like `build cache cleanup is triggered every 24 hours`?